.github/workflows/ci.yml

```diff
@@ -161,31 +161,31 @@ jobs:
           - build: linux-x86_64-gnu
             os: ubuntu-latest
             rust: stable
-            target: x86_64-unknown-linux-gnu
+            target: x86_64-linux-gnu
             cross: false
 
           - build: linux-x86_64-musl
             os: ubuntu-latest
             rust: stable
-            target: x86_64-unknown-linux-musl
+            target: x86_64-linux-musl
             cross: true
 
           - build: linux-aarch64-gnu
             os: ubuntu-latest
             rust: stable
-            target: aarch64-unknown-linux-gnu
+            target: aarch64-linux-gnu
             cross: true
 
           - build: linux-aarch64-musl
             os: ubuntu-latest
             rust: stable
-            target: aarch64-unknown-linux-musl
+            target: aarch64-linux-musl
             cross: true
 
           - build: linux-i686-gnu
             os: ubuntu-latest
             rust: stable
-            target: i686-unknown-linux-gnu
+            target: i686-linux-gnu
```

Comment:
windows is missing am I right?

Reply:
I think we don't have builds for windows in our CI. Do you want me to add that as well?

If so I would request for a new PR for that so that it won't collide with the feature and so.